### ISSUE TITLE
error dialogs don't identify the application

### ISSUE THREAD

**Issue by EdStauff** [dukus/digiCamControl#369]
I can't provide steps to reproduce the error, because it happened out of the blue:  for no apparent reason, an error dialog appeared with the window title "Application Error", and window text "Do you want to send log file to developers ?".  I had to use Process Explorer to determine that the error dialog belonged to "CameraControl.exe", which was part of digiCamControl.  A brief search of the digiCamControl source code found dozens of instances of this bug: displaying an error message without identifying the application.
You might think it would be obvious what application displayed the error dialog, but it wasn't.  I wasn't using digiCamControl, and had in fact not used it for a few days.  I don't remember what application was active when the error appeared, but it had nothing to do with digiCamControl.  It was most likely Windows Explorer, the Google Chrome browser, or Notepad++.

**Comment by dukus** [comment]
You are right, for me as developer is obvious most of the time the application running in debugger, but for simple user how run the application 1-2 times in a month isn't.

I will try to fix this kind of issues in this summer.
